daemon/socket/socket_windows.go

```diff
@@ -0,0 +1,13 @@
+// +build windows
+
+package socket
+
+import (
+	"net"
+
+	"github.com/natefinch/npipe"
+)
+
+func makeSocket(transportAddress string, groupShared bool) (net.Listener, error) {
+	return npipe.Listen(transportAddress)
```

Comment:
Perhaps we should add a comment re: windows pipe permissions and how this is only accessible to the user for future reference?